### ISSUE TITLE
Employer Search is missing profile switch demonstration with the tabs

### ISSUE THREAD

**Issue by mahestro** [topcoderinc/mjolnir-web#37]
### OS and Browser Details ###
Mac OS 10.13.3 / Chrome63

### Screen Name where the issue occurs ###
Search

### Exact Reproduction Steps ###
1. Create an employer profile or log in with user sss@sss.sss pass sss.
2. Go to search, notice tab names (friends/potential hires).
3. Go to menu (burger menu on the left), switch to a employee profile (woman).
4. Go to search, notice tab names (friends/potential hires) - should be friends/opportunities.

### Actual  Result ###
So far the profile switching is done. It's missing to display the Opportunities tab change once the profile is switched from Employer to Employee.

### Expected Result ###
From challenge: The app has slight differences depending on which profile is selected in the main menu section (https://marvelapp.com/350cda8/screen/32333155). In this instance the profile selected in the Users section of the app would be an Office profile as the toggle button says Friends and Potential Hires. This same person could switch to their individual profile and the toggle would change to Friends and Opportunities (https://marvelapp.com/350cda8/screen/33172975).

**Comment by srajas0** [comment]
@mahestro This flow cannot be implemented in prototype. We don't know which user is logged in either employee or employer? But in back end, can easily identify and easily implemented. I don't think, this implementation is here valid.

**Comment by mahestro** [comment]
I agree. If we do it here it will add a lot of - not necessary - logic that will be heavily modified in the next phase.
